convert/src/tests.rs

```diff
@@ -0,0 +1,55 @@
+use super::*;
+
+#[test]
+fn convert_to_mongo_timestamp_and_back_test() {
+    let now = std::time::UNIX_EPOCH
+        + std::time::Duration::from_secs(
+            std::time::SystemTime::now()
+                .duration_since(std::time::UNIX_EPOCH)
+                .unwrap()
+                .as_secs(),
+        ); //This is needed because we need to discard anything below the second
+    let mongo_now = mongo::systime_to_timestamp(now);
+    assert_eq!(now, mongo::timestamp_to_systime(mongo_now));
+}
+#[test]
+fn convert_to_protobuf_timestamp_and_back_test() {
+    let now = std::time::SystemTime::now();
+    let proto_now = proto::systime_to_timestamp(now);
+    assert_eq!(now, proto::timestamp_to_systime(proto_now));
+}
+
+#[test]
+fn convert_to_protobuf_timestamp_test() {
+    let epoch = std::time::UNIX_EPOCH;
```

Comment:
here and below, use a testing value a different from the unix epoch. That one is a very specific value that might be the default so your tests would pass even if the functions would return a default constructed timestamp.

Something like yesterday would work, but I would not use now() as that is going to change on every test run and you would want your tests to be as deterministic as possible

Reply:
Great idea